internal/transform/trustline.go

```diff
@@ -39,12 +39,15 @@ func TransformTrustline(ledgerChange ingest.Change, header xdr.LedgerHeaderHisto
 		return TrustlineOutput{}, errors.Wrap(err, fmt.Sprintf("could not create ledger key string for trustline with account %s and asset %s", outputAccountID, asset.ToAsset().StringCanonical()))
 	}
 
+	var assetTypeString string
```

Comment:
nit: I think you don't need to create a new variable and can just reuse `assetType` because it is already a string variable defined on line 33

In any case this is a non blocking suggestion